plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialCount.java

```diff
@@ -70,51 +76,101 @@ public static void registerTags() {
         // @mechanism MaterialTag.count
         // @group properties
         // @description
-        // Returns the minimum amount of pickles allowed in a Sea Pickle material, or eggs in a TurtleEgg material.
+        // Returns the minimum amount of pickles allowed in a Sea Pickle material, or eggs in a Turtle Egg material, or charges in a Respawn Anchor material.
         // -->
         PropertyParser.<MaterialCount>registerTag("count_min", (attribute, material) -> {
             return new ElementTag(material.getMin());
         }, "pickle_min");
 
+        // <--[tag]
+        // @attribute <MaterialTag.hatch_count>
+        // @returns ElementTag(Number)
+        // @mechanism MaterialTag.hatch_count
+        // @group properties
+        // @description
+        // Returns the amount of eggs that will hatch from a Turtle Egg material.
+        // -->
+        PropertyParser.<MaterialCount>registerTag("hatch_count", (attribute, material) -> {

```

Comment:
None of any of this belongs here.

Reply:
Shall I create a new property for hatch_count exclusively?

Reply:
No. Turtle egg 'hatch' value is already tracked by the MaterialAge property